src/pages/profile.js

```diff
@@ -37,10 +61,25 @@ const Profile = ({ data, gitHubUsername }) => {
                 <a
                   className="github-link"
                   href={`https://www.github.com/${user.nickname}`}
+                  target="_blank"
+                  rel="noreferrer"
                 >
                   <GitHubIcon className="profile-github-icon" />
-                  GitHub account | @{user ? user.nickname : ''}
+                  GitHub account
                 </a>
+                {commentsConnected ? (
+                  <a
+                    className="disqus-link"
+                    target="_blank"
+                    rel="noreferrer"
+                    href="https://disqus.com/home"
+                  >
+                    <DisqusIcon className="profile-disqus-icon" />
+                    Disqus account
+                  </a>
+                ) : (
+                  <></>
```

Comment:
Can you return nothing here instead of empty brackets or do we need the empty brackets for some reason?